module-1/lab-data_cleaning/your-code/weather.ipynb

```diff
@@ -11,38 +11,2047 @@
   },
```

Comment:
Good try Ivan. Great using plots but there are a few steps of cleaning you need to follow before analizing the dataset. I'm sending you in slack a possible solution to the additional challenge.